static/app/views/settings/project/server-side-sampling/promo.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from '@emotion/styled';
+
+import onboardingServerSideSampling from 'sentry-images/spot/onboarding-server-side-sampling.svg';
+
+import Button from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+
+import {SERVER_SIDE_DOC_LINK} from './utils';
+
+export function Promo() {
+  return (
+    <StyledEmptyStateWarning withIcon={false}>
```

Comment:
Should we use `OnboardingPanel` instead?
https://github.com/getsentry/sentry/blob/0336bd09c60cc91d2ba9d380d79eac130ac0bcc5/static/app/components/onboardingPanel.tsx